Sources/MongoSwift/ConnectionString.swift

```diff
@@ -78,6 +78,24 @@ internal class ConnectionString {
             }
         }
 
+        if let ssTimeout = options?.serverSelectionTimeoutMS {
+            guard ssTimeout > 0 && ssTimeout <= Int32.max else {
```

Comment:
maybe instead of comparing against the max and doing another cast below, we can do it in one step and just do `let timeout = Int32(exactly: ssTimeout)`

Reply:
done